docs/instruments-database/Balance Scales/Balance Scales.md

```diff
@@ -14,21 +14,23 @@ You can find all the available instruments from the sidebar
 
 ## Balance Scales 
 
- <details> 
- <summary>Category Description</summary> 
- Balances, Scales and Weighing. Laboratory balances are used to accurately determine the mass or weight of an item or substance within a specific weight range and to a particular readability. They are typically used to measure the weight of smaller amounts of substances in grams, milligrams, or micrograms.
- 
- </details> 
-
- <div className="flex flex-wrap" style={{ marginLeft: "-55px" }}>
-
-
-<div className="p-4">
-
-<a href="/instruments-database/balance-scales/mettler-toledo/mt-standard-interface-communication-software">
-<figure style={{ width: "200px", height: "200px", objectFit: "scale-down", marginRight: "15px" }}>
-<img src="https://res.cloudinary.com/dhopxs1y3/image/upload/w_600,q_auto,f_auto/e_bgremoval/v1692394030/Instruments/Balance%20Scales/MT-Standard-Interface-Communication-Software/file.jpg" style={{ width: "200px", height: "200px", objectFit: "scale-down", marginRight: "15px" }} />
-<figcaption>MT Standard Interface Communication Software</figcaption>
-</figure>
-</a></div>
+<details> 
+<summary>Category Description</summary> 
+Balances, Scales and Weighing. Laboratory balances are used to accurately determine the mass or weight of an item or substance within a specific weight range and to a particular readability. They are typically used to measure the weight of smaller amounts of substances in grams, milligrams, or micrograms.
+</details> 
+
+<!-- Custom component -->
+import AppThumbnail from '@site/src/components/AppThumbnail';
+
+<div className="flex flex-wrap">
+
+<div>
+    <AppThumbnail 
+        path='balance-scales/mettler-toledo/mt-standard-interface-communication-software'
+        img='https://res.cloudinary.com/dhopxs1y3/image/upload/w_600,q_auto,f_auto/w_600,q_auto,f_auto/e_bgremoval/v1692394030/Instruments/Balance%20Scales/MT-Standard-Interface-Communication-Software/file.jpg'
```

Comment:
img should be indented to the same level as path

I'm not sure if it's just the GitHub viewer here, but seems like there could be a lot of extraneous and inconsistent newlines and whitespace